modules/admin-credentials-root-secret-formats.adoc

```diff
@@ -63,9 +68,7 @@ stringData:
   azure_region: <Region>
 ----
 
-On Microsoft Azure, the credentials secret format includes two properties that must
-contain the cluster's infrastructure ID, generated randomly for each cluster
-installation. This value can be found after running create manifests:
+On Microsoft Azure, the credentials secret format includes two properties that must contain the cluster's infrastructure ID, generated randomly for each cluster installation. This value can be found after running create manifests:
```

Comment:
This is news to me. The Azure secret format only includes information specific to talking with Azure, and should have nothing cluster-specific. https://github.com/openshift/cloud-credential-operator#azure has an example of what would be passedthrough now that CCO literally doesn't change the Secret at all.

Reply:
Yeah - I unwrapped that text because I thought it might need a :eyes: I'll take a look at that link and see how best to revise. Is this statement inaccurate for 4.6+?

Reply:
Even looking back to 4.6, It looks like the Secret format hasn't changed at all https://github.com/openshift/cloud-credential-operator/tree/release-4.6#azure

Reply:
Resoled via Slack chat today :+1: